beamer/health/check.py

```diff
@@ -0,0 +1,514 @@
+from collections import defaultdict
+from dataclasses import dataclass, field
+from datetime import datetime
+from pathlib import Path
+from typing import DefaultDict, Optional, TypedDict, cast
+
+import toml
+import time
+from eth_typing import BlockNumber
+from typing_extensions import NotRequired
+from web3 import Web3
+from web3.constants import ADDRESS_ZERO
+from web3.middleware import geth_poa_middleware
+
+import beamer.agent.contracts
+import beamer.agent.events
+from beamer.agent.config import _merge_dicts
+from beamer.agent.events import ClaimMade, DepositWithdrawn, RequestCreated, RequestFilled
+from beamer.agent.typing import ChainId
+from beamer.agent.util import load_ERC20_abi
+from beamer.health.notify import Message, NotificationConfig, Notify, NotificationState
+from eth_utils import to_checksum_address
+
+
+class NotificationTypes:
+    REQUEST_EXPIRED = "RequestExpired"
+    DEPOSIT_WITHDRAWN = "DepositWithdrawn"
+    UNCLAIMED_FILL = "UnclaimedFill"
+    CHALLENGE_GAME = "ChallengeGame"
+    CHALLENGE_GAME_CLAIM_BY_SOMEONE_ELSE = "ChallengeGameClaimedBySomeoneElse"
```

Comment:
Not sure whether this is intentional, but I notice `CLAIM_BY` on the left and `ClaimedBy` on the right. Other types have the same expression on left and right.